engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/RuntimeServerTest.scala

```diff
@@ -108,7 +108,7 @@ class RuntimeServerTest
       def code =
         metadata.appendToCode(
           """
-            |from Standard.Base.Data.Numbers import Number
+            |from Standard.Base.Data.Numbers import    all
```

Comment:
Why have the additional indentation?
```suggestion
            |from Standard.Base.Data.Numbers import all
```

Reply:
Removing the spaces would shift offsets of the test. They are intentional.

Reply:
I see, ok